src/components/NcAppContent/NcAppContent.vue

```diff
@@ -67,9 +67,9 @@ The list size must be between the min and the max width value.
 					'app-content-wrapper--show-list': !showDetails,
 					'app-content-wrapper--mobile': isMobile,}">
 				<NcAppDetailsToggle v-if="showDetails" @click.native.stop.prevent="hideDetails" />
-				<slot v-if="!showDetails" name="list" />
+				<slot v-show="!showDetails" name="list" />
```

Comment:
ok, I understand. So it simply ignored the v-show part during my tests and worked like before 8.10.0, because hiding is done with this css, as I can see now.
```
.app-content-wrapper--no-split {
        &.app-content-wrapper--show-list :deep() {
                .app-content-list {
                        display: flex;
                }
                .app-content-details {
                        display: none;
                }               
        }
        &.app-content-wrapper--show-details :deep() {
                .app-content-list {
                        display: none;
                }               
                .app-content-details {
                        display: block;
                }
        }
}

```

But why the change to remove the list here completely?